### PR TITLE
About Box L10N

### LINKED ISSUE #11887: Localize About and Licenses dialogs

### PR DESCRIPTION
**Pull request by HansMuller**
Fixes https://github.com/flutter/flutter/issues/11887

Adds two material localizations:
```
  "aboutListTileTitle": "About $applicationName",
  "@aboutListTileTitle": {
    "description": "The default title for AboutListTile",
    "type": "text"
  },

  "licensesPagePoweredByTitle": "Powered by Flutter",
  "@licensesPagePoweredByTitle": {
    "description": "The Powered by Flutter subtitle for the [LicensePage] widget.",
    "type": "text"
  },
```

### CODE REVIEW

File: packages/flutter/lib/src/material/i18n/localizations.dart [hunk 1]
```diff
@@ -314,6 +338,8 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "nextPageTooltip": r"Nächste Seite",
     "previousPageTooltip": r"Vorherige Seite",
     "showMenuTooltip": r"Menü anzeigen",
+    "aboutListTileTitle": r"Über $applicationName",
+    "licensesPagePoweredByTitle": r"Angetrieben durch Flutter",
```
Comment (goderbauer) [inline packages/flutter/lib/src/material/i18n/localizations.dart]: That's a tough one. I don't think German has a good translation for this. In fact, most hip German apps would probably just use the english "Powered by xyz" here. If you are against using Denglish (mix of Deutsch and English), you could say something like "Mit Flutter gebaut" (which translates back to "built with Flutter").

Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/localizations.dart]: Mit Flutter gebaut sounds good to me!

**Comment by Hixie** [comment]
I think it may be better to make the entire license screen explicitly English. It needs to be LTR for the licenses anyway (right now the punctuation gets all messed up in RTL). In which case, "powered by flutter" can remain in English to avoid complicated gyrations with the directionality.

**Comment by cbracken** [comment]
LGTM for Japanese and French!

File: packages/flutter/lib/src/material/i18n/material_zh.arb [hunk 2]
```diff
@@ -7,6 +7,7 @@
   "nextPageTooltip": "下一页",
   "previousPageTooltip": "上一页",
   "showMenuTooltip": "显示菜单",
+  "aboutListTileTitle": "关于$applicationName",
```
Comment (xster) [inline packages/flutter/lib/src/material/i18n/material_zh.arb]: LG

File: packages/flutter/lib/src/material/material_localizations.dart [hunk 3]
```diff
@@ -41,6 +41,9 @@ abstract class MaterialLocalizations {
   /// The default [PopupMenuButton] tooltip.
   String get showMenuTooltip;
 
+  /// The default title for AboutListTile.
```
Comment (Hixie) [inline packages/flutter/lib/src/material/material_localizations.dart]: hyperlink

File: packages/flutter/lib/src/material/i18n/material_en.arb [hunk 4]
```diff
@@ -47,6 +47,12 @@
     "type": "text"
   },
 
+  "aboutListTileTitle": "About $applicationName",
+  "@aboutListTileTitle": {
+    "description": "The default title for AboutListTile",
+    "type": "text"
+  },
```
Comment (Hixie) [inline packages/flutter/lib/src/material/i18n/material_en.arb]: do we document the magic substitution strings?

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/lib/src/material/i18n/material_en.arb (hunk 4 above)
Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/material_en.arb]: There is some info about resource ids and the details about how the *.arb files are mapped to a per locale table of resourceId to value tables in dev/tools/gen_localizations.dart.

I need to add an i18n/README.md that explains what's going on more clearly.

Comment (HansMuller) [reply packages/flutter/lib/src/material/i18n/material_en.arb]: https://github.com/flutter/flutter/issues/12113

**Pull request merged**